vignettes/SingleR.Rmd

```diff
@@ -225,8 +226,10 @@ hESCs <- hESCs[common,]
 hESCs <- logNormCounts(hESCs)
 ```
 
-The reference data sets all come with two sets of cell labels, `label.main` and `label.fine`. 
-The "main" labels tend to be less fine-grained than the "fine" labels, e.g., instead of specifying different subsets of T cells, the main labels will lump them all together under the label "T cells". 
+The reference data sets all come with two sets of cell labels: `label.main` and `label.fine`. 
+The "main" labels tend to be less fine-grained than the "fine" labels, e.g. instead of specifying different subsets of T cells, the main labels will lump them all together under the broader label "T cells".
+For more details about the individual cell type labels available for the different reference data sets, see the vignette `r Biocpkg("SingleR", vignette = "ReferenceDataSets")`.
```

Comment:
if you try running this, you will see why I used `label=` in the previous call. But you don't even need this, because this information is already mentioned in the previous link above.

Reply:
I'm a big proponent of repeating information. I prefer having the same info in multiple places. Repetition is helpful.

Reply:
That's a fine sentiment, up to the point where you have to remember to update all the repeated links when the URL changes.

In any case, that still isn't quite right, because the vignette argument needs to be `datasets.html` (as this will be the base name of the URL once it is compiled) and the `labels=` argument needs to refer to the plain English text of the link.